sqlx/lexer.ts

```diff
@@ -20,7 +22,6 @@ const SQL_LEXER_TOKEN_NAMES = {
   SINGLE_QUOTE_STRING: LEXER_STATE_NAMES.SQL + "_singleQuoteString",
```

Comment:
shouldn't we be removing these two token types now? same with inner_sql?

Reply:
Ah good point